MediaBrowser.Model/Configuration/LibraryOptions.cs

```diff
@@ -80,6 +81,7 @@ public LibraryOptions()
         public bool RequirePerfectSubtitleMatch { get; set; }
 
         public bool SaveSubtitlesWithMedia { get; set; }
+        public bool AutoCollection { get; set; }
```

Comment:
I still believe a more descriptive name should be used. You have to keep in mind that this will be written to an XML file and it might not be clear what AutoCollection means.

Reply:
What do you think of `AutoAddToCollection`? Is it better?

Reply:
`AutomaticallyAddToCollection` seems fine to me